---------------------
PatchSet 323 
Date: 2010/09/28 05:25:26
Author: millert
Branch: HEAD
Tag: (none) 
Log:
Undo last change now that RCS keyword expansion is disabled on the perl sources

Members: 
	cpan/ExtUtils-MakeMaker/t/parse_version.t:1.3->1.4 
	cpan/Module-Build/t/metadata.t:1.3->1.4 
	cpan/Module-Build/t/moduleinfo.t:1.3->1.4 

Index: src/gnu/usr.bin/perl/cpan/ExtUtils-MakeMaker/t/parse_version.t
diff -u src/gnu/usr.bin/perl/cpan/ExtUtils-MakeMaker/t/parse_version.t:1.3 src/gnu/usr.bin/perl/cpan/ExtUtils-MakeMaker/t/parse_version.t:1.4
--- cpan/ExtUtils-MakeMaker/t/parse_version.t:1.3	Sat Sep 25 16:34:33 2010
+++ cpan/ExtUtils-MakeMaker/t/parse_version.t	Tue Sep 28 12:25:26 2010
@@ -12,7 +12,7 @@
 
 my %versions = (q[$VERSION = '1.00']            => '1.00',
                 q[*VERSION = \'1.01']           => '1.01',
-                q[($VERSION) = q/Revision: 32208 / =~ /(\d+)/g;] => 32208,
+                q[($VERSION) = q$Revision: 32208 $ =~ /(\d+)/g;] => 32208,
                 q[$FOO::VERSION = '1.10';]      => '1.10',
                 q[*FOO::VERSION = \'1.11';]     => '1.11',
                 '$VERSION = 0.02'               => 0.02,
@@ -34,8 +34,8 @@
                 qq[\$Something::VERSION == 1.0\n\$VERSION = 2.3\n]                     => '2.3',
                 qq[\$Something::VERSION == 1.0\n\$VERSION = 2.3\n\$VERSION = 4.5\n]    => '2.3',
 
-                '$VERSION = sprintf("%d.%03d", q/Revision: 3.74 / =~ /(\d+)\.(\d+)/);' => '3.074',
-                '$VERSION = substr(q/Revision: 2.8 /, 10) + 2 . "";'                   => '4.8',
+                '$VERSION = sprintf("%d.%03d", q$Revision: 3.74 $ =~ /(\d+)\.(\d+)/);' => '3.074',
+                '$VERSION = substr(q$Revision: 2.8 $, 10) + 2 . "";'                   => '4.8',
 
                );
 
Index: src/gnu/usr.bin/perl/cpan/Module-Build/t/metadata.t
diff -u src/gnu/usr.bin/perl/cpan/Module-Build/t/metadata.t:1.3 src/gnu/usr.bin/perl/cpan/Module-Build/t/metadata.t:1.4
--- cpan/Module-Build/t/metadata.t:1.3	Sat Sep 25 16:34:33 2010
+++ cpan/Module-Build/t/metadata.t	Tue Sep 28 12:25:26 2010
@@ -171,9 +171,9 @@
 {
   $dist->change_file( 'lib/Simple.pm', <<'---' );
 package Simple;
-$VERSION = version->new('0.60.' . (qw/Revision: 128 /)[1]);
+$VERSION = version->new('0.60.' . (qw$Revision: 128 $)[1]);
 package Simple::Simon;
-$VERSION = version->new('0.61.' . (qw/Revision: 129 /)[1]);
+$VERSION = version->new('0.61.' . (qw$Revision: 129 $)[1]);
 ---
   $dist->regen;
   my $provides = new_build()->get_metadata()->{provides};
Index: src/gnu/usr.bin/perl/cpan/Module-Build/t/moduleinfo.t
diff -u src/gnu/usr.bin/perl/cpan/Module-Build/t/moduleinfo.t:1.3 src/gnu/usr.bin/perl/cpan/Module-Build/t/moduleinfo.t:1.4
--- cpan/Module-Build/t/moduleinfo.t:1.3	Sat Sep 25 16:34:33 2010
+++ cpan/Module-Build/t/moduleinfo.t	Tue Sep 28 12:25:26 2010
@@ -424,9 +424,9 @@
   # Make sure we handle version.pm $VERSIONs well
   $dist->change_file( 'lib/Simple.pm', <<'---' );
 package Simple;
-$VERSION = version->new('0.60.' . (qw/Revision: 128 /)[1]);
+$VERSION = version->new('0.60.' . (qw$Revision: 128 $)[1]);
 package Simple::Simon;
-$VERSION = version->new('0.61.' . (qw/Revision: 129 /)[1]);
+$VERSION = version->new('0.61.' . (qw$Revision: 129 $)[1]);
 ---
   $dist->regen;
 
